From 017a53fca98e90e12f45ffe419ff60476e6d02b1 Mon Sep 17 00:00:00 2001
From: "James E. Blair" <jeblair@hp.com>
Date: Mon, 5 Dec 2011 11:23:40 -0800
Subject: [PATCH] Rename .glance-venv to .venv.

This simplifies a number of Jenkins jobs which currently, other
than directory names, could be the same for all OpenStack
projects.  By renaming the virtualenv directory, the redundant
Jenkins virtualenv build and copy jobs can be eliminated.

(cherry picked from commit 37fc2b00c0a8dd67221d83697609913aa78d89c7)

Change-Id: I7916783d863e5184bad8293c3aed5ba119e374a9
---
 .bzrignore            |    1 +
 .gitignore            |    1 +
 .mailmap              |    2 ++
 Authors               |    1 +
 run_tests.sh          |    2 +-
 tools/install_venv.py |    4 ++--
 tools/with_venv.sh    |    2 +-
 7 files changed, 9 insertions(+), 4 deletions(-)

diff --git a/.bzrignore b/.bzrignore
index 0dc1ca0..3c39f0b 100644
diff --git a/.gitignore b/.gitignore
index 410f8fc..b181715 100644
diff --git a/.mailmap b/.mailmap
index 0293555..8467e9b 100644
diff --git a/Authors b/Authors
index 36234ce..2e8e0b7 100644
diff --git a/run_tests.sh b/run_tests.sh
index eec2ac1..e1fbb7b 100755
--- a/run_tests.sh
+++ b/run_tests.sh
@@ -29,7 +29,7 @@ function process_option {
   esac
 }
 
-venv=.glance-venv
+venv=.venv
 with_venv=tools/with_venv.sh
 always_venv=0
 never_venv=0
diff --git a/tools/install_venv.py b/tools/install_venv.py
index 5c0f71a..7a9473a 100644
--- a/tools/install_venv.py
+++ b/tools/install_venv.py
@@ -28,7 +28,7 @@ import sys
 
 
 ROOT = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
-VENV = os.path.join(ROOT, '.glance-venv')
+VENV = os.path.join(ROOT, '.venv')
 PIP_REQUIRES = os.path.join(ROOT, 'tools', 'pip-requires')
 
 
@@ -124,7 +124,7 @@ def print_help():
  To activate the Glance virtualenv for the extent of your current shell session
  you can run:
 
- $ source .glance-venv/bin/activate
+ $ source .venv/bin/activate
 
  Or, if you prefer, you can run commands in the virtualenv on a case by case
  basis by running:
diff --git a/tools/with_venv.sh b/tools/with_venv.sh
index fc5ed77..c8d2940 100755
--- a/tools/with_venv.sh
+++ b/tools/with_venv.sh
@@ -1,4 +1,4 @@
 #!/bin/bash
 TOOLS=`dirname $0`
-VENV=$TOOLS/../.glance-venv
+VENV=$TOOLS/../.venv
 source $VENV/bin/activate && $@
-- 
1.7.6.5

